app/models/receptacle.rb

```diff
@@ -95,9 +95,14 @@ def any_barcode_matching?(other_barcode)
   has_many :submitted_assets, foreign_key: :asset_id # Created to associate an asset with an order
   has_many :orders, through: :submitted_assets
   has_many :ordered_studies, through: :orders, source: :study
+  has_many :direct_submissions, ->() { distinct }, through: :orders, source: :submission
 
   has_many :tags, through: :aliquots
 
+  # Historically this used to be the only way to jump up to the submission without
+  # going via stock-wells. However, newer aliquots are associated with the request.
+  # This newer approach better handles cross-submission pooling, such as that used
+  # in the GBS pipelines.
```

Comment:
Did you want to reference the `direct_submissions` association in this comment, as the implementation of the 'newer approach' that you mention?

Reply:
Actually no, direct submissions is different, but I should probably make it clearer what the newer approach is. I'll also look at adding a story to backfill aliquot request data, as then we can simplify a lot of this.